src/views/App/App.scss

```diff
@@ -1,33 +1,225 @@
+.ah-jumbotron {
+    background-color: $ah-secondary-color;
+    padding: 0px;
+}
+
+// Navbar
+.ah-jumbotron h1,
+.navbar-brand {
+    font-size: 2.5rem;
+    font-family: "Vidaloka", serif;
+}
+
+.modal-title {
+    font-family: "Vidaloka", serif;
+    font-size: 2rem;
+    line-height: 0.9;
+}
+
+// Twitter
+.fa-twitter {
+    color: #1da1f2;
+}
+
+// Facebook
+.fa-facebook {
+    color: #3b5998;
+}
+
+// Google
+.fa-google {
+    color: #ea4335;
+}
+
+.fa-facebook:hover,
+.fa-twitter:hover,
+.fa-envelope:hover,
+.fa-flag:hover {
+    opacity: 0.7;
+    cursor: pointer;
+}
+
+.form-group input:focus,
+textarea:focus {
+    background-color: $bg-color;
+}
+
+.custom-file-label {
+    background-color: transparent;
+}
+
 .App {
-  text-align: center;
+    text-align: center;
 }
 
 .App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
+    animation: App-logo-spin infinite 20s linear;
+    height: 40vmin;
 }
 
 $white: #FFFFFF;
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: $white;
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: $white;
 }
 
 .App-link {
-  color: #61dafb;
+    color: #61dafb;
 }
 
 @keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+    from {
+        transform: rotate(0deg);
+    }
+    to {
+        transform: rotate(360deg);
+    }
+}
+
+.ah-facebook-btn {
+    background-color: $facebook-color;
+    color: #ffffff !important;
+}
+
+.ah-twitter-btn {
+    background-color: $twitter-color;
+    color: #ffffff !important;
+}
+
+.ah-google-btn {
+    background-color: $google-color;
+    color: #ffffff !important;
+}
+
+.modal-body .ah-err-msg {
+    color: red;
+}
+
+#snackbar {
+    visibility: hidden;
+    /* Hidden by default. Visible on click */
+    min-width: 250px;
+    /* Set a default minimum width */
+    margin-left: -125px;
+    /* Divide value of min-width by 2 */
+    background-color: #333;
+    /* Black background color */
+    color: #fff;
+    /* White text color */
+    text-align: center;
+    /* Centered text */
+    border-radius: 2px;
+    /* Rounded borders */
+    padding: 16px;
+    /* Padding */
+    position: fixed;
+    /* Sit on top of the screen */
+    z-index: 1;
+    /* Add a z-index if needed */
+    left: 50%;
+    /* Center the snackbar */
+    bottom: 30px;
+    /* 30px from the bottom */
 }
+
+
+/* Show the snackbar when clicking on a button (class added with JavaScript) */
+
+#snackbar.show {
+    visibility: visible;
+    /* Show the snackbar */
+    /* Add animation: Take 0.5 seconds to fade in and out the snackbar.
+ However, delay the fade out process for 2.5 seconds */
```

Comment:
Please get rid of the comments, keep your code clean

Reply:
Done